### ISSUE TITLE
Inconsistent Record Character Limits

### ISSUE THREAD

**Issue by Mwahahahaha** [Aurorastation/Aurora.3#5766]
In the character setup, you can input information on your records for up to 3072 characters.
However, in the round, if you use a records console and click on the appropriate Important Notes section (where the character setup information will be), and a new dialog box opens up containing the setup information which can now be modified, clicking "OK" may cut it down as the maximum length for this dialog box entry is 1024 characters.

**Sanitize Proc for reference**
code\\__helpers\text.dm
![image](https://user-images.githubusercontent.com/3950115/49989714-208c5a80-ffb6-11e8-8a71-d58c5af7b730.png)
(uses MAX_MESSAGE_LEN as the default value if var/max_length is not specified when used)

**1024 characters for security consoles**
code\game\machinery\computer\security.dm
![image](https://user-images.githubusercontent.com/3950115/49989736-30a43a00-ffb6-11e8-8c66-69985ff39fe2.png)
(does not specify another value for var/max_length; uses MAX_MESSAGE_LEN)

**3072 characters for setup**
code\modules\client\preference_setup\general\05_background.dm
![image](https://user-images.githubusercontent.com/3950115/49989782-4dd90880-ffb6-11e8-879c-f4d572f5290d.png)
(specifies another value for var/max_length: MAX_PAPER_MESSAGE_LEN)

**Definitions for reference**
code\\__defines\misc.dm
![image](https://user-images.githubusercontent.com/3950115/49989831-6a754080-ffb6-11e8-930e-065c93b7dfdb.png)

**Comment by Sindorman** [comment]
Wow, that is an extensive explanations.

TL;DR: security console uses wrong define as max length - easy fix.
